eng/helix/Helix.proj

```diff
@@ -113,6 +113,14 @@
     <HelixTargetQueue Include="@(HelixAvailableTargetQueue)" />
   </ItemGroup>
 
+  <!-- Additional package feeds -->
+  <ItemGroup>
+    <AdditionalDotNetPackageFeed Include="https://dotnetbuilds.blob.core.windows.net/internal"
```

Comment:
Conditionally include if DotNetBuildsInternalContainerReadToken is not empty.

Reply:
Actually, this is not needed since internal builds will always have this token and it's already conditioned to only be included for internal builds.